skrifa/src/provider.rs

```diff
@@ -29,6 +32,12 @@ pub trait MetadataProvider<'a>: Sized {
     /// given informational string identifier.
     fn localized_strings(&self, id: StringId) -> LocalizedStrings<'a>;
 
+    fn best_family_name(&self) -> Option<String>;
+
+    fn best_subfamily_name(&self) -> Option<String>;
+
+    fn best_full_name(&self) -> Option<String>;
+
```

Comment:
these would deserve docs explaining what 'best' means :)

Reply:
You're right. I missed that. I added it. Thanks a lot for your valid feedback.